INFO     pages\create_store.py:create_store.py:59 Request data: {'id': 'store.uid', 'petId': 124721, 'quantity': 7380, 'shipDate': '1980-09-12T00:50:07', 'status': 'approved', 'complete': True}
INFO     src\utils\http_methods.py:http_methods.py:49 Response data: {'id': 'store.uid', 'petId': 124721, 'quantity': 7380, 'shipDate': '1980-09-12T00:50:07', 'status': 'approved', 'complete': True}
ERROR    src\utils\schemas\store\store_validator:store_validator.py:22 Invalid data format [{"type":"missing","loc":["status"],"msg":"Field required","input":{"code":400,"message":"Input error: unable to convert input to io.swagger.petstore.model.Order"},"url":"https://errors.pydantic.dev/2.4/v/missing"},{"type":"missing","loc":["complete"],"msg":"Field required","input":{"code":400,"message":"Input error: unable to convert input to io.swagger.petstore.model.Order"},"url":"https://errors.pydantic.dev/2.4/v/missing"}]